### DISCUSSION TITLE
termux build apk ( ndk for c, go )

### DISCUSSION THREAD

**Discussion by MatejMagat305** [termux/termux-app#2339] [Q&A] [answered]
halo, I m very often in termux and I m often using golang with library: https://github.com/fyne-io/fyne, it can also build android(apk), but it has to have ndk is there any posibilities to install ndk for it? (on proot linux library work as well as apk(from computer), but I want compile apk on termux.........., at least .so library, because other I can create manualy or copy..............)

**Comment by agnostic-apollo** [comment, marked as answer]
https://github.com/termux/termux-packages/pull/7227#issuecomment-893022283

**Reply by MatejMagat305** [reply]
so thank, all is clear except where I take jdk?

**Reply by agnostic-apollo** [reply]
`pkg install openjdk-17`

**Reply by MatejMagat305** [reply]
thank

**Reply by agnostic-apollo** [reply]
welcome
